cocoatouch/src/main/java/org/robovm/apple/uikit/UIApplication.java

```diff
@@ -359,9 +360,37 @@ public void invoke(NSNotification a) {
             }
         });
         
+        registerCustomIBClasses();
+        
         main(argc, argv, principalClassName, delegateClassName);
     }
     
+    /**
+     * Registers all custom interface builder classes from the compiler generated ib.classes file, if any.
+     */
+    private static void registerCustomIBClasses() {
+        File ibClassesFile = new File(NSBundle.getMainBundle().getBundlePath() + "/ib.classes");
+        if (ibClassesFile.exists()) {
+            try {
+                BufferedReader reader = new BufferedReader(new FileReader(ibClassesFile));
```

Comment:
Sorry, missed this one the first time I had a look. try-with-resources again. :-)